content/docs/rendering-elements.md

```diff
@@ -1,75 +1,77 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: Rendering av element
 permalink: docs/rendering-elements.html
 redirect_from:
   - "docs/displaying-data.html"
 prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+Element är de minsta byggstenarna i React-applikationer.
 
-An element describes what you want to see on the screen:
+Ett element beskriver vad som skall visas på skärmen:
 
 ```js
 const element = <h1>Hello, world</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+Till skillnad från DOM-elementen i en webbläsare är React-element enkla och kostnadseffektiva objekt. React DOM tar hand om uppdatering av DOM och tillhörande React-element.
 
->**Note:**
+
+>**Obs:**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+>Man kan förväxla element med det mer kända begreppet "komponenter". Vi kommer att introducera komponenter i [nästa avsnitt](/docs/components-and-props.html). Elementen utgör grunden för komponenterna, så vi rekommenderar dig att läsa detta avsnitt innan du går vidare.
```

Comment:
'One' could be translated straight to 'En' here.

Reply:
Fixing. 👍

Reply:
I'm pretty sure "en" is still a regional expression, there may be people who never see/use this (I can't speak for them though, because I belong to the region who's using it 😄)

Reply:
Should I change it back to "Man" or something else?

Reply:
Yes, I prefer "man", I think it looks less like slang. In my case, when I read "en", my mind immediately starts reading with an accent :smile:

Jokes aside (even if it's true), put simply I think it's easier and less risk to use "man" (thinking about future translations and consistency too. "man" is very common). 

But just my 2 cents of course! You guys do as you see fit, since we are split anyways. Perhaps @rloqvist has anything to add

Reply:
The reason I suggested 'En' instead of 'Man' is not dialectal, but rather because it is gender neutral.